base/deprecated.jl

```diff
@@ -865,4 +865,118 @@ for f in (:sin, :sinh, :sind, :asin, :asinh, :asind,
+            :airy, :airyx, :besselj0, :besselj1, :bessely0, :bessely1, # base/special/bessel.jl
+        :cbrt, :sinh, :cosh, :tanh, :atan, :asinh, :exp, :erf, :erfc, :exp2, # base/math.jl
+            :expm1, :exp10, :sin, :cos, :tan, :asin, :acos, :acosh, :atanh, # base/math.jl
+            #=:log,=# :log2, :log10, :lgamma, #=:log1p,=# :sqrt,            # base/math.jl
+        :abs, :abs2, :angle, :isnan, :isinf, :isfinite, # base/floatfuncs.jl
+        :acos_fast, :acosh_fast, :angle_fast, :asin_fast, :asinh_fast, # base/fastmath.jl
+            :atan_fast, :atanh_fast, :cbrt_fast, :cis_fast, :cos_fast, # base/fastmath.jl
+            :cosh_fast, :exp10_fast, :exp2_fast, :exp_fast, :expm1_fast, # base/fastmath.jl
+            :lgamma_fast, :log10_fast, :log1p_fast, :log2_fast, :log_fast, # base/fastmath.jl
+            :sin_fast, :sinh_fast, :sqrt_fast, :tan_fast, :tanh_fast, # base/fastmath.jl
+        :cis, # base/complex.jl
+        )
+    @eval @dep_vectorize_1arg Number $f
+end
+for f in (
+        :sind, :cosd, :tand, :asind, :acosd, :atand, :asecd, :acscd, :acotd, # base/special/trig.jl
+        :invdigamma, # base/special/gamma.jl
+        :erfinc, :erfcinv, # base/special/erf.jl
+        :rad2deg, :deg2rad, :exponent, :significand, # base/math.jl
+        :trunc, :floor, :ceil, :round, # base/floatfuncs.jl
+        :unix2datetime, :rata2datetime, :julian2datetime, # base/dates/conversions.jl
+        )
+    @eval @dep_vectorize_1arg Real $f
+end
+# base/complex.jl
+@dep_vectorize_1arg Complex round
+@dep_vectorize_1arg Complex float
+for f in (
+        :year, :month, :day, :week, :dayofmonth, :yearmonth, :monthday, :yearmonthday, # base/dates/accessors.jl
+        :firstdayofweek, :lastdayofweek, :firstdayofmonth, # base/dates/adjusters.jl
+            :lastdayofmonth, :firstdayofyear, :lastdayofyear, # base/dates/adjusters.jl
+            :firstdayofquarter, :lastdayofquarter, # base/dates/adjusters.jl
+        :dayname, :dayabbr, :dayofweek, :dayofweekofmonth, # base/dates/query.jl
+            :daysofweekinmonth, :monthname, :monthabbr, :daysinmonth, # base/dates/query.jl
+            :isleapyear, :dayofyear, :daysinyear, :quarterofyear, :dayofquarter, # base/dates/query.jl
+    )
+    @eval @dep_vectorize_1arg Dates.TimeType $f
+end
+for f in (
+    :hour, :minute, :second, :millisecond, # base/dates/accessors.jl
+    :Date, :datetime2unix, :datetime2rata, :datetime2julian, # base/dates/conversions.jl
+    )
+    @eval @dep_vectorize_1arg Dates.DateTime $f
+end
+@dep_vectorize_1arg Dates.Date Datetime # base/dates/conversions.jl
+
+# Deprecate @vectorize_2arg-vectorized functions from...
+for f in (
+        :polygamma, :zeta, :beta, :lbeta, # base/special/gamma.jl
+        :airy, :airyx, :besseli, :besselix, :besselj, :besseljx, # base/special/bessel.jl
+            :besselk, :besselkx, :bessely, :besselyx, :besselh, # base/special/bessel.jl
```

Comment:
Yes, exactly :). The third line (beginning `:besselk`, thrice indented) is a continuation of the second line's list (beginning `:airy`, twice indented), the functions on the second and third lines coming from the same file (base/special/bessel.jl per the trailing comments). The same extra-indentation-for-list-continuation occurs in the earlier loops over functions as well, reflecting the file grouping of functions. If you prefer removal of the extra indentation indicating file grouping, I am happy to change that uniformly. Thanks!